Ruins/src/main/java/atomicstryker/ruins/common/RuinTemplateRule.java

```diff
@@ -88,7 +88,7 @@ public RuinTemplateRule(PrintWriter dpw, RuinTemplate r, String rule, boolean de
             throw new Exception("No blockIDs specified for rule [" + rule + "] in template " + owner.getName());
         }
         condition = Integer.parseInt(blockRules[0]);
-        chance = Integer.parseInt(blockRules[1]);
+        chance = Math.min(Math.max(Double.parseDouble(blockRules[1])/100, 0), 1);

```

Comment:
Again, 0 chance ... does that work?

Reply:
Zero works here, too. It's always been a valid value and has the expected effect--effectively rendering the rule a no-op (or preserveBlock, in Ruins terms). The only change is I'm clamping the value to be between 0% and 100%, inclusive (though values outside that range don't have the same deleterious effects as invalid weights).

Reply:
I did test these, by the way, if that's what you're asking.

Reply:
Good good, i didn't want to risk a zero division somewhere.